src/pages/dataTableUtilities/actions.js

```diff
@@ -7,6 +7,10 @@ import { parsePositiveInteger } from '../../utilities';
 import Settings from '../../settings/MobileAppSettings';
 import { SETTINGS_KEYS } from '../../settings/index';
 
+import { MODAL_KEYS } from '../../utilities/getModalTitle';
+
+const { STOCKTAKE_OUTDATED_ITEM, EDIT_STOCKTAKE_BATCH, STOCKTAKE_COMMENT_EDIT } = MODAL_KEYS;
```

Comment:
It may be easier not to destructure these, just in case there are more in future. Referencing them as `MODAL_KEYS.STOCKTAKE_OUTDATED_ITEM` is ok. I mention because this line has been used in a number of files already